### PR TITLE
Upgrade/package version

### PR DESCRIPTION
**Pull request by MapleWithoutWords**

### CODE REVIEW

**Comment by doddgu** [comment]
@capdiem @zhenlei520 PTAL

File: src/Web/Masa.Framework.Admin.Web/Pages/App/Invoice/Add.razor [hunk 1]
```diff
@@ -218,7 +218,7 @@
             <MSelect BackgroundColor="white" Items=_payments ItemValue="u => u" Dense Label="Accept Payment Via" ItemText="u => u" Outlined @bind-Value=_payment></MSelect>
             <MSwitch Value=true Class="mt-n3" Color="#7367f0" Label="Payment Terms"></MSwitch>
             <MSwitch Value=true Class="mt-n3" Color="#7367f0" Label="Client Notes"></MSwitch>
-            <MSwitch Class="mt-n3" Color="#7367f0" Label="Payment Stub"></MSwitch>
+            <MSwitch Value=false Class="mt-n3" Color="#7367f0" Label="Payment Stub"></MSwitch>
```
Comment (capdiem) [inline src/Web/Masa.Framework.Admin.Web/Pages/App/Invoice/Add.razor]: 只是演示的代码吗？`Value=true/false`说实话没什么用，都改成 `TValue="bool"`。

File: src/Web/Masa.Framework.Admin.Web/Pages/App/Todo/TodoDetail.razor.cs [hunk 2]
```diff
@@ -77,11 +77,11 @@ protected override async Task OnParametersSetAsync()
 
         if (ValueChanged.HasDelegate && !Value && _mForm != null)
         {
-            await _mForm.ResetValidationAsync();
+            _mForm.Reset();
```
Comment (capdiem) [inline src/Web/Masa.Framework.Admin.Web/Pages/App/Todo/TodoDetail.razor.cs]: ResetValidation

File: src/Web/Masa.Framework.Admin.Web/Pages/App/User/Edit.razor [hunk 3]
```diff
@@ -161,9 +161,9 @@
                     <MCol Md=4 Sm=12>
                         <span class="text-btn">Contact Options</span>
                         <div class="d-flex mt-n2">
-                            <MCheckbox Label="Email" HideDetails="@("auto")" Color="primary"></MCheckbox>
-                            <MCheckbox Label="Message" HideDetails="@("auto")" Color="primary" Class="ml-2"></MCheckbox>
-                            <MCheckbox Label="Phone" HideDetails="@("auto")" Color="primary" Class="ml-2"></MCheckbox>
+                            <MCheckbox Label="Email" HideDetails="@("auto")" Color="primary" Value="false"></MCheckbox>
+                            <MCheckbox Label="Message" HideDetails="@("auto")" Color="primary" Class="ml-2" Value="false"></MCheckbox>
+                            <MCheckbox Label="Phone" HideDetails="@("auto")" Color="primary" Class="ml-2" Value="false"></MCheckbox>
```
Comment (capdiem) [inline src/Web/Masa.Framework.Admin.Web/Pages/App/User/Edit.razor]: 不是加`Value=false`，是加`TValue="bool"`

File: src/Web/Masa.Framework.Admin.Web/Pages/Authentication/Components/Register.razor [hunk 4]
```diff
@@ -35,7 +35,7 @@
                     Outlined
                     Dense></MTextField>
 
-        <MCheckbox Class="mt-0 pt-0" Color="#7367f0" Label="I agree to privacy policy & terms"></MCheckbox>
+        <MCheckbox Value=false Class="mt-0 pt-0" Color="#7367f0" Label="I agree to privacy policy & terms"></MCheckbox>
```
Comment (capdiem) [inline src/Web/Masa.Framework.Admin.Web/Pages/Authentication/Components/Register.razor]: `TValue=bool`

File: src/Web/Masa.Framework.Admin.Web/Pages/Others/AccountSettings/AccountSettings.razor [hunk 5]
```diff
@@ -200,13 +200,13 @@
...
+                <MSwitch Value="false" Label="Email me when someone comments on my article"></MSwitch>
+                <MSwitch Value="false" Label="Email me when someone answers on my form"></MSwitch>
+                <MSwitch Value="false" Label="Email me hen someone follows me"></MSwitch>
                 <p class="grey--text">APPLICATION</p>
-                <MSwitch Label="News and announcements"></MSwitch>
-                <MSwitch Label="Weekly product updates"></MSwitch>
-                <MSwitch Label="Weekly blog digest"></MSwitch>
+                <MSwitch Value="false" Label="News and announcements"></MSwitch>
+                <MSwitch Value="false" Label="Weekly product updates"></MSwitch>
+                <MSwitch Value="false" Label="Weekly blog digest"></MSwitch>
```
Comment (capdiem) [inline src/Web/Masa.Framework.Admin.Web/Pages/Others/AccountSettings/AccountSettings.razor]: `TValue=bool`

**Review by capdiem — CHANGES_REQUESTED** [review]

File: sdks/MASA.Framework.Sdks.Authentication/MASA.Framework.Sdks.Authentication.csproj [hunk 6]
```diff
@@ -7,9 +7,10 @@
   </PropertyGroup>
   
   <ItemGroup>
-    <PackageReference Include="Masa.Utils.Caller.HttpClient" Version="$(MasaUtilsPackageVersion)" />
+    <PackageReference Include="Masa.Contrib.Service.Caller.HttpClient" Version="$(MasaUtilsPackageVersion)" />
```
Comment (zhenlei520) [inline sdks/MASA.Framework.Sdks.Authentication/MASA.Framework.Sdks.Authentication.csproj]: Version = "$(MasaContribPackageVersion)"

File: sdks/MASA.Framework.Sdks.Authentication/_Imports.cs [hunk 7]
```diff
@@ -12,8 +12,7 @@
 global using Masa.Framework.Sdks.Authentication.Response.Configuration;
 global using Masa.Framework.Sdks.Authentication.Response.LogStatistics;
 global using Masa.Framework.Sdks.Authentication.Response.Users;
-global using Masa.Utils.Caller.HttpClient;
-global using Masa.Utils.Exceptions;
+global using Masa.Contrib.Service.Caller.HttpClient;
```
Comment (zhenlei520) [inline sdks/MASA.Framework.Sdks.Authentication/_Imports.cs]: global using sort

File: Directory.Build.props [hunk 8]
```diff
@@ -1,9 +1,9 @@
 <Project>
   <PropertyGroup>
     <MicrosoftPackageVersion>6.0.0</MicrosoftPackageVersion>
-    <MasaUtilsPackageVersion>0.5.0</MasaUtilsPackageVersion>
-    <MasaContribPackageVersion>0.5.0</MasaContribPackageVersion>
-    <MasaBlazorPackageVersion>0.5.0</MasaBlazorPackageVersion>
+    <MasaUtilsPackageVersion>1.0.0-rc.1</MasaUtilsPackageVersion>
```
Comment (zhenlei520) [inline Directory.Build.props]: Utils and Contrib share the same version, which can be controlled by a parameter, and subsequent versions also use the same parameter to make it easier to understand

File: src/Services/Masa.Framework.Admin.Service.Authentication/Masa.Framework.Admin.Service.Authentication.csproj [hunk 9]
```diff
@@ -10,11 +10,11 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <PackageReference Include="Masa.Utils.Caching.DistributedMemory" Version="$(MasaUtilsPackageVersion)" />
-      <PackageReference Include="Masa.Utils.Caching.Redis" Version="$(MasaUtilsPackageVersion)" />
-      <PackageReference Include="Masa.Utils.Development.Dapr.AspNetCore" Version="$(MasaUtilsPackageVersion)" />
+      <PackageReference Include="Masa.Contrib.Caching.MultilevelCache" Version="$(MasaUtilsPackageVersion)" />
```
[the remark this answers is not in the document]
Comment (zhenlei520) [reply src/Services/Masa.Framework.Admin.Service.Authentication/Masa.Framework.Admin.Service.Authentication.csproj:13]: The same is true below

File: src/Services/Masa.Framework.Admin.Service.Authentication/Program.cs [hunk 10]
```diff
@@ -1,3 +1,5 @@
+using Masa.BuildingBlocks.Dispatcher.IntegrationEvents;
```
Comment (zhenlei520) [inline src/Services/Masa.Framework.Admin.Service.Authentication/Program.cs]: use global using

File: src/Services/Masa.Framework.Admin.Service.LogStatistics/_Imports.cs [hunk 11]
```diff
@@ -39,3 +39,4 @@
 global using Quartz.Impl;
 global using Quartz.Spi;
 global using System.Reflection;
+global using Masa.BuildingBlocks.Dispatcher.IntegrationEvents;
```
Comment (zhenlei520) [inline src/Services/Masa.Framework.Admin.Service.LogStatistics/_Imports.cs:42]: use global using

File: src/Services/Masa.Framework.Admin.Service.User/_Imports.cs [hunk 12]
```diff
@@ -60,3 +60,4 @@
 global using System.Reflection;
 global using System.Security.Claims;
 global using System.Text;
+global using Masa.BuildingBlocks.Dispatcher.IntegrationEvents;
```
Comment (zhenlei520) [inline src/Services/Masa.Framework.Admin.Service.User/_Imports.cs]: use global using

File: src/Web/Masa.Framework.Admin.Web/_Imports.cs [hunk 13]
```diff
@@ -21,7 +21,7 @@
 global using Masa.Framework.Sdks.Authentication.Internal;
 global using Masa.Framework.Sdks.Authentication.Internal.Enum;
 global using Masa.Framework.Sdks.Authentication.Request.LogStatistics;
-global using Masa.Utils.Caller.Core;
+global using Masa.Contrib.Service.Caller;
```
Comment (zhenlei520) [inline src/Web/Masa.Framework.Admin.Web/_Imports.cs]: use global using

**Review by zhenlei520 — CHANGES_REQUESTED** [review]

File: sdks/MASA.Framework.Sdks.Authentication/MASA.Framework.Sdks.Authentication.csproj (hunk 6 above)
Comment (zhenlei520) [reply sdks/MASA.Framework.Sdks.Authentication/MASA.Framework.Sdks.Authentication.csproj]: The same is true below, It is recommended that Masa Framework use the same parameter to control the version

File: src/Services/Masa.Framework.Admin.Service.Authentication/Program.cs [hunk 14]
```diff
@@ -6,9 +8,7 @@
 
 #endif
 
-builder.Services
-    .AddMasaRedisCache(builder.Configuration.GetSection("RedisConfig"))
-    .AddMasaMemoryCache();
+builder.Services.AddDistributedCache(options=>options.UseStackExchangeRedisCache());
```
Comment (zhenlei520) [inline src/Services/Masa.Framework.Admin.Service.Authentication/Program.cs]: Multi-level caching is required here

File: src/Web/Masa.Framework.Admin.Web/Program.cs [hunk 15]
```diff
@@ -4,21 +4,13 @@
...
 builder.Services.AddServerSideBlazor();
-builder.Services.AddMasaBlazor(builder =>
-{
-    builder.UseTheme(option =>
-        {
-            option.Primary = "#4318FF";
-            option.Accent = "#4318FF";
-        }
-    );
-}).AddGlobalForServer();
```
Comment (capdiem) [inline src/Web/Masa.Framework.Admin.Web/Program.cs]: Pro的配色不能删

**Review by capdiem — CHANGES_REQUESTED** [review]

**Review by zhenlei520 — APPROVED** [review]
LGTM @capdiem

**Review by capdiem — APPROVED** [review]
LGTM!

**Review by capdiem — APPROVED** [review]

**Review by capdiem — APPROVED** [review]

**Review by MayueCif — APPROVED** [review]

**Pull request merged**